website/docusaurus.config.js

```diff
@@ -46,6 +46,15 @@ module.exports = {
   organizationName: 'litmuschaos',
   projectName: 'litmus',
   themeConfig: {
+    algolia: {
+      apiKey: '7af78d639b7342f7da38c5c5c51993b0',
+      indexName: 'litmuschaos',
+      contextualSearch: true
+
+      // appId: 'YOUR_APP_ID',
```

Comment:
can you remove these comments...and this the ok to make the API key public or does it have no real impact if leaked?

Reply:
It won't have any impact.